Rakefile

```diff
@@ -459,7 +459,7 @@ def confirm_branch_name(name)
 end
 
 def each_project_with_common_build(&b)
-  except = %w[ rspec ]
+  except = %w[ rspec rspec-rails ]
```

Comment:
This will also prevent the tasks working on markdown files, we need to refactor this so its flagged or passed in from the travis task only...